ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -177,6 +177,9 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
         
     
         // Push
+        case "push#getRegistrationToken":
+            return try await AirshipProxy.shared.push.getRegistrationToken()
```

Comment:
I think the await should be omitted here ?

Reply:
I don't think so cause it returns a Promise